README.md

```diff
@@ -13,39 +13,119 @@ Artifacts in support of ODH Edge use cases that show integration with Red Hat Ad
+
+In [pipelines/README.md](pipelines/README.md) we show how to take the trained models,
+store them in a S3 bucket,
+build container image(s) with the model and serving runtime using OpenShift Pipelines,
+push the container image(s) to an image registry accessible by the near edge cluster(s),
+and update a clone of this repository with a pull request,
+configuring `acm/odh-edge/apps/*/kustomization.yaml` with location and digest (SHA-256) of the built images.
+
+You can skip this step if you do not wish to rebuild the container images.
+If you use the default configuration as shown in this git repository,
+you will use already built container images from the https://quay.io/organization/rhoai-edge repositories.
+
+### Serving to near edge remote clusters
+
+We assume that you have an admin access to an OpenShift cluster where you can install and configure
+the Red Hat Advanced Cluster Management operator and which will serve as the central ACM hub,
+and other OpenShift cluster(s) with admin privileges that you will import to the ACM hub cluster
+and that will play the role of near edge location(s), managed from the central ACM hub.
+
+#### ACM setup
+
+The following setup steps need to be done as an admin user with `cluster-admins` privileges.
+We will describe how to do them using OpenShift Console but there may be other ways to achieve the same results.
+
+1. On the ACM hub cluster, install the Advanced Cluster Management for Kubernetes operator.
+   In OpenShift Console of the cluster, find and run the installation of the operator
+   in Menu > Operators > OperatorHub.
+1. Create the MultiClusterHub.
+   In OpenShift Console in Menu > Operators > Installed Operators > Advanced Cluster Management for Kubernetes >
+   the MultiClusterHub tab, use the "Create MultiClusterHub" button to start the installation.
+1. Refresh the Web OpenShift Console.
+1. Create a cluster set for your near edge clusters.
+   In OpenShift Console it can be done in All Clusters menu > Infrastructure > Clusters > Cluster sets tab.
+   This repository assumes name `poc-near-edge` in `clusterSets`
+   in [acm/registration/near-edge/base/near-edge.yaml](acm/registration/near-edge/base/near-edge.yaml)
+   so update that YAML file if you use different name.
+1. Import the near edge cluster(s).
+   In OpenShift Console it can be done in All Clusters menu > Infrastructure > Clusters > Cluster list tab.
+   In the Cluster set popup menu, select the cluster set you created for the near edge clusters.
+   If you use the Import mode of "Enter your server URL and API token for the existing cluster",
+   obtain the address of the server and the token on the other cluster's OpenShift Console 
+   at the top-right corner user menu > Copy login command, for an admin user.
+
+#### Create namespaces for applications
+
+For each of the deployed models-turned-into-application, we will create separate namespace/project on the ACM hub cluster.
```

Comment:
> "models-turned-into-application"

Are you sure this is a good term to use?

```suggestion
For each of the deployed models-turned-into-application, we will create separate namespace(s)/project(s) on the ACM hub cluster.
```

Reply:
> > "models-turned-into-application"
> Are you sure this is a good term to use?

Well, the application term is overloaded and has a lot of specific meanings in the ACM world, so I wanted something to express that I talk about our specific instances.

What would you suggest instead?

Reply:
As for the plurals, isn't the point here that for each application, there is one namespace (project)?